[openvpn] Add pmacct json-log audit support (#3686)
deckhouse/deckhouse: modules/500-openvpn/docs/README.md

- a route to the local network;
- a route to the cluster service network;
- a route to the Pod network.

## User's traffic audit

> The module can log VPN connections in JSON format. Traffic is grouped
> by `src_ip, dst_ip, src_port, dst_port, ip_proto` fields. Container logs can be collected and sent for storage or analysis
> using the [log-shipper](../460-log-shipper/) module.

Reviewer: ```suggestion
The module can log user's activity via VPN in JSON format. Traffic is grouped
by `src_ip, dst_ip, src_port, dst_port, ip_proto` fields. Container logs can be collected and sent to storage for further audit
using the [log-shipper](../460-log-shipper/) module.
```
